doc/scripts/genrest.py

```diff
@@ -80,6 +80,8 @@ def expr_str(expr):
 def write_kconfig_rst():
     # The "main" function. Writes index.rst and the symbol RST files.
 
+    # accelerare doc building by skipping kconfig option documentation.
+    TURBO_MODE = True if 'KCONFIG_DOC_TURBO_MODE' in os.environ else False
```

Comment:
I think this will run in TURBO_MODE if KCONFIG_DOC_TURBO_MODE has any value, right?